src/Framework/Framework/Controls/AlternateCultureLinks.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using DotVVM.Framework.Configuration;
+using DotVVM.Framework.Hosting;
+using DotVVM.Framework.Routing;
+
+namespace DotVVM.Framework.Controls
+{
+    public class AlternateCultureLinks : CompositeControl 
```

Comment:
I'd include a doccoment with a reference to the Google/MDN documentation for link rel=alternate